storybook/storybook-react/src/components/Checkbox/Checkbox.docs.mdx

```diff
@@ -0,0 +1,17 @@
+import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import * as CheckboxStories from "./Checkbox.stories.tsx";
+import README from "../../../../../packages/css/src/checkbox/README.md?raw";
+
+<Meta of={CheckboxStories} />
+
+<Markdown>{README}</Markdown>
+
+<Primary />
+
+<Controls />
```

Comment:
Like you mentioned in @VincentSmedinga changes in the no marker PR, I think that these controls are in the storybook example for MDX but feel double/unnecessary because the default variant (story) will have the same controls and they distract from the actual docs.

Reply:
The `Controls` component is used here to add controls to the default variant? But let's discuss this in a seperate PR

Reply:
No these controls add knobs to the docs story, the stories for each variant contain the controls. You don't really need controls here